src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -145,45 +159,65 @@ def generate_MCP_1_default_config(self, filename):
                                                                  'test_user_1')
     # Define DPAs
     dpa_1 = {
-       'dpaId': 'east_dpa_4',
-       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+        'dpaId': 'east_dpa_4',
+        'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
     }
     dpa_2 = {
-       'dpaId': 'east_dpa_5',
-       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+        'dpaId': 'east_dpa_5',
+        'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
     }
     dpa_3 = {
-       'dpaId': 'east_dpa_6',
-       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+        'dpaId': 'east_dpa_6',
+        'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+    }
+    dpa_generic = {
+      'east_dpa_4': {
+        'points_builder': 'default (25, 10, 10, 10)',
+        'movelistMargin': 10
+      },
+      'east_dpa_5': {
+        'points_builder': 'default (25, 10, 10, 10)',
+        'movelistMargin': 10
+      },
+      'east_dpa_6': {
+        'points_builder': 'default (25, 10, 10, 10)',
+        'movelistMargin': 10
+      }
     }
 
     # Registration and grant records for multiple iterations
     cbsd_records_iteration_0_domain_proxy_0 = {
         'registrationRequests': [device_1, device_2],
-        'grantRequests': [grant_request_1, grant_request_2]
+        'grantRequests': [grant_request_1, grant_request_2],
+        'conditionalRegistrationData': [conditionals_device_2]
     }
     cbsd_records_iteration_0_domain_proxy_1 = {
         'registrationRequests': [device_3],
-        'grantRequests': [grant_request_3]
+        'grantRequests': [grant_request_3],
+        'conditionalRegistrationData': []
     }
     cbsd_records_iteration_1_domain_proxy_0 = {
         'registrationRequests': [device_4],
-        'grantRequests': [grant_request_4]
+        'grantRequests': [grant_request_4],
+        'conditionalRegistrationData': [conditionals_device_4]
```

Comment:
why this different from line 197 and 207 ?

Reply:
What difference do you see?

Reply:
This CBSD had conditional registration data and the others didn't.

Reply:
ok, got it. Thanks.